timescaledb/overview/release-notes/index.md

```diff
@@ -32,22 +32,50 @@ If you have questions about distributed hypertables, join our #multinode channel
 on [community Slack](https://slack.timescale.com/) for installation details and
 follow these [setup instructions][distributed-hypertables-setup].
 
-### What's new in TimescaleDB 2.6.1:
+### What's new in TimescaleDB 2.7:
 
-- Continuous aggregates with compression
-- `time_bucket_ng` support for N months, years and timezones on continuous aggregates
 
-You can read more about this release on our [blog post](https://tsdb.co/timescaledb-2-6).
-This release also contains bug fixes since the 2.6.0 release.
+You can recreate a continuous aggregate to benefit from the improvement, if the
+original data already exists.
 
+- Improved continuous aggregates performances and lifted several limitations (by removing reaggregation)
+
+Because continuous aggregates no longer store partials, and re-aggregation is removed, 
```

Comment:
```suggestion
This new continuous aggregate format is faster than the previous version and also
because continuous aggregates no longer store partials, and re-aggregation is removed, 
```